starport-core/src/main/scala/com/krux/starport/util/ErrorHandler.scala

```diff
@@ -82,8 +82,10 @@ object ErrorHandler extends Logging with WaitForIt {
    * @return the SES send ID
    */
   def cleanupActivityFailed(pipeline: Pipeline, stackTrace: Array[StackTraceElement]): String = {
+    val conf = StarportSettings()
```

Comment:
this may load multiple potentially inconsistent configs. You can simply change the method to the following, and conf will be available and is from the one initialized by the process

```scala
def cleanupActivityFailed(pipeline: Pipeline, stackTrace: Array[StackTraceElement])(implicit conf: StarportSettings)
```

Reply:
sweet!